templates/vars/aks/prod.json

```diff
@@ -3,19 +3,19 @@
   "contentVersion": "1.0.0.0",
   "parameters": {
     "kubernetesVersion": {
-      "value": "1.17.9"
+      "value": "1.18.8"
     },
     "nodeMinCount": {
       "value": 32
```

Comment:
Should we have min count as 32?

Reply:
changed it min:30 and max:50 as per our discussion